tests/unit/test_pair_counts.py

```diff
@@ -0,0 +1,112 @@
+from unittest import TestCase
+
+import responses
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+from tamr_unify_client.models.operation import Operation
+from tamr_unify_client.models.project.estimated_pair_counts import EstimatedPairCounts
+from tamr_unify_client.models.project.mastering import MasteringProject
+
+
+class TestPairCounts(TestCase):
+    def setUp(self):
+        auth = UsernamePasswordAuth("username", "password")
+        self.unify = Client(auth)
+
+    @responses.activate
+    def test_get(self):
+        p = MasteringProject(self.unify, self._project_json)
+        responses.add(
+            responses.GET, f"{self._url_base}/{self._api_path}", json=self._project_json
+        )
+        generated = p.estimate_pairs()
+
+        created = EstimatedPairCounts.from_json(
+            self.unify, self._estimate_json, self._api_path
+        )
+        self.assertEqual(print(generated), print(created))
```

Comment:
I'm pretty sure you want `repr()` here instead of `print()`

Reply:
true